iModelCore/ECPresentation/PublicAPI/ECPresentation/NavNodeKey.h

```diff
@@ -44,8 +44,9 @@ struct EXPORT_VTABLE_ATTRIBUTE NavNodeKey : RefCountedBase
 
 protected:
     ECPRESENTATION_EXPORT NavNodeKey(NavNodeKey const& other);
-    ECPRESENTATION_EXPORT NavNodeKey(Utf8String type, Utf8String specificationIdentifier, bvector<Utf8String> hashPath);
-    ECPRESENTATION_EXPORT ~NavNodeKey();
+    ECPRESENTATION_EXPORT NavNodeKey(Utf8String type, Utf8String specificationIdentifier, bvector<Utf8String> hashPath)
+        : m_type(type), m_specificationIdentifier(specificationIdentifier), m_hashPath(hashPath) {}
+    ECPRESENTATION_EXPORT ~NavNodeKey() {}

```

Comment:
`ECPRESENTATION_EXPORT` should not be added to inlined functions. Also, no point in keeping the empty destructor.

Reply:
Removed both.